plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -190,6 +190,13 @@ object StmtConversionVisitor : FirVisitor<ExpEmbedding, StmtConversionContext<Re
             else -> {
                 val retType = implicitInvokeCall.calleeCallableSymbol.resolvedReturnType
                 val args = implicitInvokeCall.functionCallArguments.map(data::convert)
+                // the first argument is the callee and is not considered leaking
+                args.drop(1)
+                    .filter { it.type is UnspecifiedFunctionTypeEmbedding }
```

Comment:
Nit: There's also a "filter by type" function of some kind that only gives you those elements that are of this type, but I think here there's no benefit to using it.

Reply:
Ah, that's a good point.